### PR TITLE
Adds an adminverb to write map files.

### PR DESCRIPTION
**Pull request by PJB3005**
Map files are stored in `data/logs/saved_maps`, as such they can be retried with `.getserverlog`, however the admin that started a write will also get the file sent to him with the `ftp()` proc.

**Note:** map files exported still need a _lot_ of cleaning afterwards (literally every turf has redundant variables that might even break things written to them).

### CODE REVIEW

**Comment by ghost** [comment]
:+1:

**Comment by gbasood** [comment]
:+1:

File: code/modules/awaymissions/maploader/writer.dm [hunk 1]
```diff
@@ -23,7 +17,9 @@
 	var/list/blacklist = list(
 			/atom/movable/lighting_overlay,
 			/obj/effect/beam,
-			/obj/item/projectile
+			/obj/item/projectile,
+			/mob/dview,
```
Comment (clusterfack) [inline code/modules/awaymissions/maploader/writer.dm]: Why not just all /mobs, and all /obj/effects

Comment (PJB3005) [inline code/modules/awaymissions/maploader/writer.dm]: Monkeys and shit. 

Obj/effect may be a good idea though.

**Comment by PJB3005** [comment]
Wow GitHub mobile lied to me, and said it didn't  conflict.

**Comment by PJB3005** [comment]
There we go finally fixed.

**Pull request merged**